### ISSUE TITLE
Bot won't continue after "Wohnung genauer anschauen"

### ISSUE THREAD

**Issue by Ruperr** [neopostmodern/wohnungsbot#195]
If the bot finds a suitable apartment, it sometimes stops responding, doesn't continue with the next step. Currently, it stucks at "Wohnung genauer anschauen", or on the further steps. Ideally, if it can reach the contact form, the bot get stucks on the old message, and nothing further occurs. Eventually, a "Timed out" messages appears in the menu. I've tried various settings, reset the bot, restarted it, toggled the sort by new apartments on and off, but nothing resolves the issue. Then there are times when the bot works flawlessly, only to stop working again later. Bot version 1.7.0 same problem on windows and linux.

**Comment by DeNyMus1NeBiLon** [comment]
+1 on ver. 1.7.0 on macOS

**Comment by Chris17121** [comment]
I have the same problem. The bot stucks at "Wohnung genauer anschauen"
1.7.0 on windows

**Comment by paulschraven** [comment]
Did anyone find a way to fix / circumvent this?

**Comment by paulschraven** [comment]
For me it breaks on "Wohnung 149504170 suchen" sometimes already

**Comment by DeNyMus1NeBiLon** [comment]
it's just a suspicion but i think it could have something to do with "Nur exklusiv für MieterPlus". reset your bots and pay close attention where the bot stops working, for me it's when the bot detects the above!

**Comment by Arsen0897** [comment]
I have the same problem The bot hangs on "Wohnung genauer anschauen"

**Comment by R2myz** [comment]
Same issue with "Wohnung genauer anschauen".

**Comment by Barcode133** [comment]
Same issue, trying since yesterday - it always gets stuck on "Wohnung [...] genauer anschauen" and haven't been able to get past that point at all. Tried different options, reset, APP restart, PC restart and different PC's. Issue seems to be not connected to "MieterPlus" offers, i removed districts until the bot found a flat without "Nur exklusiv für MieterPlus" as the first valid result, same issue. 
Version 1.7.0 on Windows 11 Pro and Home.

**Comment by hc975** [comment]
+1

**Comment by Nassauerland** [comment]
+1 exactly the same problem. Doesn't matter what I try it always stops at "Wohnung genauer anschauen" :c

**Comment by neopostmodern** [comment]
I just pushed a fix to the `dev` branch. Once the build action is finished I'll check if there is something you can just download, otherwise you'll have to run from source unfortunately. No guarantees on stability of the `dev` branch though.

**Comment by neopostmodern** [comment]
Actually I'm intending to release a 1.7.1 hotfix for this. Can you please test the artifacts from [the hotfix branch build](https://github.com/neopostmodern/wohnungsbot/actions/runs/8388937564) once they're ready and confirm it solves the issue for you? (You need to be logged in to download them.)

**Comment by Ruperr** [comment]
For me it doesn't seem to fill out the contact form (timed out)

**Comment by neopostmodern** [comment]
But it got past the previous issue of "genauer anschauen"?
Then I guess we're facing another issue.

**Comment by dahoo** [comment]
It is timing out for me in the contact form when there are no contact data fields and you can just apply with the profile. I solved it by changing [the selector](https://github.com/neopostmodern/wohnungsbot/blob/850c2780ee5ad4eebbe3d2338bf2a5d1def76ef3/app/utils/performApplication.js#L45) to `#contactForm-Message`.

**Comment by neopostmodern** [comment]
@dahoo Could you post the ID of the flat? I think I've never seen such an insert before.

**Comment by neopostmodern** [comment]
The issues described by @paulschraven, @Ruperr (in https://github.com/neopostmodern/wohnungsbot/issues/195#issuecomment-2015342194, sounds like #190) and @dahoo seem to be separate from this – please open new issues (if applicable) with information how to reproduce.

**Comment by neopostmodern** [comment]
Released 1.7.1

**Comment by wobe11** [comment]
Hi, I am having the same problem, using 1.8.0 under Windows 11.

**Comment by Dokrzz** [comment]
Also having the same problem @neopostmodern, will reopen the issue.

System:

- Model Name: Mac mini
- Chip: Apple M2
- System Version: macOS 15.4 (24E248)
- App Version: Wohnungsbot-1.8.0-arm64.dmg
